code/modules/research/items/design_disk.dm

```diff
@@ -6,7 +6,8 @@
 	item_state = "card-id"
 	w_class = WEIGHT_CLASS_SMALL
 	materials_base = list(MAT_STEEL = 30, MAT_GLASS = 10)
-	var/design_id
+	var/list/design_ids
+	var/design_capacity = 12
```

Comment:
i would prefer if getters/setters/helpers were used; having external access to internal state on these datastructure-like items is not a good pattern imo

this is a nitpick though it's fine for now, when we redo r&d we can worry about it

Reply:
only 1 obj can touch them atm which i why i elected not to use getters/setters
OST they really should be usable in lathes asp so making it more generic with helpers would be good

Reply:
done